modules/highgui/cmake/init.cmake

```diff
@@ -39,6 +39,13 @@ endmacro()
 add_backend("gtk" WITH_GTK)
 add_backend("win32ui" WITH_WIN32UI)
 add_backend("wayland" WITH_WAYLAND)
+if(WITH_FRAMEBUFFER AND WITH_FRAMEBUFFER_XVFB)
+  add_backend("framebuffer" WITH_FRAMEBUFFER)
+else()
+  add_backend("framebuffer" WITH_FRAMEBUFFER)
+  add_backend("framebuffer" WITH_FRAMEBUFFER_XVFB)
+endif()
```

Comment:
In our case, the user can have the value of two options WITH_FRAMEBUFFER AND WITH_FRAMEBUFFER_XVFB.
By the condition below I am trying to avoid re-include:

if(WITH_FRAMEBUFFER AND WITH_FRAMEBUFFER_XVFB)
  add_backend("framebuffer" WITH_FRAMEBUFFER)
else()
  add_backend("framebuffer" WITH_FRAMEBUFFER)
  add_backend("framebuffer" WITH_FRAMEBUFFER_XVFB)
endif()

Do I understand correctly that detect_framebuffer.cmake should be divided into two files - standard and XVFB?